tests/snapshot_tests/test_snapshots.py

```diff
@@ -406,3 +406,13 @@ def test_fr_margins(snap_compare):
 def test_scroll_visible(snap_compare):
     # https://github.com/Textualize/textual/issues/2181
     assert snap_compare(SNAPSHOT_APPS_DIR / "scroll_visible.py", press=["t"])
+
+
+def test_scroll_to_center(snap_compare):
```

Comment:
If this test is expected to fail, can you mark is as an `xfail` ?

Reply:
This test is not expected to fail _now_. It will fail eventually, when the bug is fixed.